cdc/log.cc

```diff
@@ -722,8 +728,20 @@ cdc::cdc_service::impl::augment_mutation_call(lowres_clock::time_point timeout,
 
             transformer trans(_ctxt, s);
 
+            std::vector<mutation> muts;
```

Comment:
This and next 4 lines are leftovers of previous version of the code and should be removed.